src/cfnlint/data/ExtendedSpecs/cn-northwest-1/05_pricing_property_values.json

```diff
@@ -1 +1,56 @@
-[]
```

Comment:
instance type pricing information finally working for AWS China regions: https://github.com/aws-cloudformation/cfn-python-lint/pull/1535